docs/BR.md

```diff
@@ -1739,142 +1737,875 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+
+  * CA Certificates
+    * [Section 7.1.2.1.- Root CA Certificate Profile](#7121-root-ca-certificate-profile)
+    * Subordinate CA Certificates
+      * Cross Certificates
+        * [Section 7.1.2.2 - Cross-Certified Subordinate CA Certificate Profile](#7122-cross-certified-subordinate-ca-certificate-profile)
+      * Technically Constrained CA Certificates
+        * [Section 7.1.2.3 - Technically-Constrained Non-TLS Subordinate CA Certificate Profile](#7123-technically-constrained-non-tls-subordinate-ca-certificate-profile)
+        * [Section 7.1.2.4 - Technically-Constrained TLS Subordinate CA Certificate Profile](#7124-technically-constrained-tls-subordinate-ca-certificate-profile)
+      * [Section 7.1.2.5 - TLS Subordinate CA Certificate Profile](#7125-tls-subordinate-ca-certificate-profile)
+  * [Section 7.1.2.6 - Subscriber (End-Entity) Certificate Profile](#7126-subscriber-server-certificate-profile)
+    * Domain Validated
+    * Individual Validated
+    * Organization Validated
+    * Extended Validation
+  * __**TBD: Infrastructure Certificates**__
+    * [Section 7.1.2.7 - OCSP Responder Certificate Profile](#7127-ocsp-responder-certificate-profile)
+    * __**TBD: Infrastructure Cert**__
+    * __**TBD: Precert Signing CA**__
+    * __**TBD: Interoperability Testing Certs (valid/revoked/expired)?**__
+
+#### 7.1.2.1 Root CA Certificate Profile
+
+| __Field__                      | __Description__ |
+| ---                            | ------          |
+| `tbsCertificate`               | |
+| \ \ \ \ `version`              | MUST be v3(2) |
+| \ \ \ \ `serialNumber`         | MUST be a non-sequential number greater than zero (0) containing at least 64 bits of output from a CSPRNG. |
+| \ \ \ \ `signature`            | See [Section 7.1.3.2](#7132-signature-algorithmidentifier) |
+| \ \ \ \ `issuer`               | Encoded value MUST be byte-for-byte identical to the encoded `subject` |
+| \ \ \ \ `validity`             | __**TBD**__ |
+| \ \ \ \ `subject`              | See [Section 7.1.2.8.1](#71281-ca-certificate-naming) |
+| \ \ \ \ `subjectPublicKeyInfo` | See [Section 7.1.3.1](#7131-subjectpublickeyinfo) |
+| \ \ \ \ `issuerUniqueID`       | MUST NOT be present |
+| \ \ \ \ `subjectUniqueID`      | MUST NOT be present |
+| \ \ \ \ `extensions`           | See [Section 7.1.2.1.1](#71211-root-ca-extensions) |
+| `signatureAlgorithm`           | Encoded value MUST be byte-for-byte identical to the `tbsCertificate.signature`. |
+| `signature`                    | |
+
+##### 7.1.2.1.1 Root CA Extensions
+
+| __Extension__                     | __Presence__ | __Critical__ | __Description__ |
+| ----                              | -            | -            | ----- |
+| `authorityKeyIdentifier`          | SHOULD       | N            | See [Section 7.1.2.1.2](#71212-authority-key-identifier) |
```

Comment:
Unless I'm missing something, this is a MAY-level requirement in 5280 as it serves no purpose to be included in a root certificate.

Reply:
It’s relevant for several client implementations, most importantly Windows, as part of the signal that a certificate is self-signed and self-issued. While I think that’s a poor design, this resolves that ambiguity and is a potential MUST in the future.